dynamiqs/time_array.py

```diff
@@ -535,3 +548,29 @@ def __add__(self, other: ArrayLike | TimeArray) -> TimeArray:
             return SummedTimeArray([*self.timearrays, other])
         else:
             return NotImplemented
+
+
+def eval_timearrays_sum(timearrays: [TimeArray], t: float) -> TimeArray:
+    return jtu.tree_reduce(
+        jnp.add,
+        jtu.tree_map(
+            lambda x: x(t), timearrays, is_leaf=lambda x: isinstance(x, TimeArray)
+        ),
+    )
```

Comment:
Why not move this function directly within the `__call__` method? Is it used somewhere else?

Reply:
It used to but not anymore, I'll fix it